sdk/ml/azure-ai-ml/azure/ai/ml/operations/_batch_endpoint_operations.py

```diff
@@ -238,6 +239,10 @@ def invoke(
             self._validate_deployment_name(endpoint_name, deployment_name)
 
         if input and isinstance(input, Input):
+            warnings.warn(
+                "the 'input' parameter for batch endpoint invoke is deprecated. Please use 'inputs' instead.",
```

Comment:
nit: Can we change it to be sentence case?
Also, do we need to add some documentation link of how to use `inputs`?

Reply:
Sentence syntax has been changed to match the deprecation format. However, it is not standard practice to add links to the deprecation warning.